src/styles/stars.scss

```diff
@@ -0,0 +1,28 @@
+.card__rating-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 400;
+  font-size: 10px;
+  margin-bottom: 24px;
```

Comment:
it belongs to card block, so move it there

Reply:
done